webapp/src/ts/modules/reports/reports.component.ts

```diff
@@ -26,6 +26,7 @@ import { TranslateService } from '@mm-services/translate.service';
 import { ReportsSidebarFilterComponent } from '@mm-modules/reports/reports-sidebar-filter.component';
 import { AuthService } from '@mm-services/auth.service';
 import { OLD_REPORTS_FILTER_PERMISSION } from '@mm-modules/reports/reports-filters.component';
+import {UserContactService} from '@mm-services/user-contact.service';
```

Comment:
```suggestion
import { UserContactService } from '@mm-services/user-contact.service';
```
I think the IDE should have settings for all this. If you're using WebStorm then: Preferences > code style > javascript or typescript > Spaces. I'm using Webstorm and it adds the imports with the space around it